packages/app/src/cli/services/extensions/common.ts

```diff
@@ -59,14 +59,9 @@ export async function canEnablePreviewMode({
   token: string
   apiKey: string
 }) {
+  if (localApp.allExtensions.length > 0) return true
   const {dashboardManagedExtensionRegistrations} = (await fetchAppExtensionRegistrations({token, apiKey})).app
   if (dashboardManagedExtensionRegistrations.length > 0) return true
-  const themeExtensions = localApp.allExtensions.filter((ext) => ext.isThemeExtension)
-  if (themeExtensions.length > 0) return true
-
-  const unifiedDeployment = remoteApp?.betas?.unifiedAppDeployment ?? false
-  const draftableExtensions = localApp.allExtensions.filter((ext) => ext.isDraftable(unifiedDeployment))
-  if (unifiedDeployment && draftableExtensions.length > 0) return true
```

Comment:
Do theme extensions still need to be filtered out here?

Reply:
Yeah I'll bring back isDraftable